### DISCUSSION TITLE
養兒防老，養老防兒

### DISCUSSION THREAD

**Discussion by WeiYen1103** [1101-GS4719-C/Python-Community-GS4719-C#1760] [General]
今天滑fb時，看到一個影片，讓我印象深刻，
畫面中是一個蹲坐在地上的老人，骨瘦如材，是那種只會在畫作中看過的模樣，我沒想過人類真的可以瘦到只看的到骨頭，
他是被自己的兒子兒媳從家中硬生生敢出來的老人，蹲坐在路邊已經三天了，靠著別人施捨的食物才得以留有一口氣，
最近流行起一句話：養兒防老，養老防兒
這不僅讓我深思，到底是一個什麼樣的存在，會讓人願意花自己的大半輩子，自己的錢財，去陪伴去養大一個人，
那想必是愛吧，那我們能確保我們的孩子對我們是否也有堅定不移的愛嗎，
如果是我，我會先被好一筆錢，給自己養老，有能力才生小孩，以免造成自己或是這個孩子的負擔，
也不會出現影片中那般被丟棄在路邊的模樣。

**Comment by davidteng00000** [comment]
影片呈現的情況令人心痛。養兒防老，養老防兒，反映出家庭與愛的關係。確保互相尊重和愛護是至關重要的，但也需平衡責任與自主權。考慮未來是明智之舉，以免後悔。

**Comment by eric202501** [comment]
「養兒防老，養老防兒」這句話反映了一種現實考量，同時也提醒我們應該要謹慎對待家庭關係和人生的選擇，才不會對自己感到後悔

**Comment by 111426027** [comment]
我覺得就算是親身骨肉還是沒有辦法真的指望他們可以在你老的時候照顧你，所以自己存到養老金可以照顧自己才是根本的解決之道。

**Comment by jim1743** [comment]
我覺得未來若我要養小孩，我必定要有一定的經濟能力，畢竟如果我連自己都養不活了，我為什麼要拖一個無辜的生命過上和我一樣的生活，並且和孩子陪伴也要一定的心力和愛，如果這些我都沒辦法給他，那他將來憑什麼要養我。

**Comment by EPHMR0524** [comment]
老人被棄養確實是一件令人心痛的事情，這無疑是一場悲劇。
但即使有這樣的影片在網路上面流傳，我對養兒防老還是樂觀的，畢竟如果自己的孩子是在一個充滿愛的環境長大的話，那應該也不至於老了的時候被小孩子拋棄，這當然是建立在小孩沒有走偏的情況就是了，但孩子走歪有一部份也是父母的責任。

**Comment by YOGA012** [comment]
我非常認同你的觀點。影片中老人被遺棄的情景令人心碎，也讓我們反思養兒防老這一傳統觀念。養大孩子本應是出於愛，但卻無法保證孩子們能同樣回報這份愛。因此，確保自己有足夠的資金來養老是明智的選擇，不僅可以避免成為孩子的負擔，也能確保自己在年老時能過上有尊嚴的生活。現代社會變遷迅速，養老方式也需要與時俱進，這樣才能避免影片中的悲劇重演。

**Comment by YuanWang08** [comment]
心疼那些遭受虐待和忽視的老人，他們在人生的晚年觀察到別人賤視他們的生命尊嚴。他們所經歷的，讓我們思考家庭、親情與尊嚴的意義。將權利和責任合理地平衡是非常重要的。我們確實需要規劃未來，以確保自己在老年時得到應有的照顧，而不是完全依賴子女

**Comment by sandyho0307** [comment]
我覺得依賴子女來確保晚年的生活可能是不可靠的，因此，為自己的退休生活做好充分的財務規劃顯得尤為重要，確保自己在老年時有足夠的經濟支持，可以獨立於子女生活。

**Comment by cassy0201** [comment]
我覺得這同時也顯現了家庭教育和成長環境的重要性

**Comment by buoioiry** [comment]
我覺得要養孩子的前提要是自己有足夠的經濟能力養孩子以外也要有自己的生活，有些人生了小孩就沒有自己平時的休閒娛樂，也有些人養小孩的方式就只是讓他長大而已，但我認為比較好的狀態應該是生了小孩以後可以讓他跟自己都在健康快樂的環境成長。

**Comment by OHoln** [comment]
我自始至終認為養兒防老是一個很荒唐的說法，如果今天我自身的經濟能力只能做到勉強養大我的小孩，甚至以後要靠他養，我根本不可能選擇生小孩，畢竟他沒有權力決定自己要不要出生，抱有養兒防老這種觀念是非常自私的

**Comment by 710388** [comment]
這是什麼奇怪的說法，感覺太重視利益了，小孩還是要更重視感情才是，不要當作保險
